backend/src/main/java/ai/verta/modeldb/dataset/DatasetDAORdbImpl.java

```diff
@@ -65,7 +65,7 @@
 
   // Queries
   private static final String GET_DATASET_BY_IDS_QUERY =
-      "From DatasetEntity ds where ds.id IN (:ids)";
+      "From DatasetEntity ds where ds.id IN (:ids) AND ds.deleted = false ";
```

Comment:
Should we use `ModelDBConstants.DELETED` here too?

Reply:
updated with constant name